src/app/pages/CreateConnectorPage/StepReview.tsx

```diff
@@ -161,91 +160,92 @@ export const StepReviewComponent: React.FC<StepReviewComponentProps> = ({
         />
       }
     >
-      <Form>
-        {savingError && (
-          <FormAlert>
-            <Alert
-              variant="danger"
-              title={savingError}
-              aria-live="polite"
-              isInline
-            />
-          </FormAlert>
-        )}
-        {toggleView ? (
-          <ViewJSONFormat
-            kafka={kafka}
-            namespace={namespace}
-            connectorType={connectorType}
-            name={name}
-            topic={topic}
-            userErrorHandler={userErrorHandler}
-            userServiceAccount={userServiceAccount}
-            configString={configString}
-            configurationSteps={configurationSteps}
+      {savingError && (
+        <FormAlert>
+          <Alert
+            variant="danger"
+            title={savingError}
+            aria-live="polite"
+            isInline
           />
-        ) : (
-          <>
-            <Grid>
-              <GridItem span={4}>
-                <strong>{t('connectorCategory')}</strong>
-              </GridItem>
-              <GridItem span={8}>
+        </FormAlert>
+      )}
+      {toggleView ? (
+        <ViewJSONFormat
+          kafka={kafka}
+          namespace={namespace}
+          connectorType={connectorType}
+          name={name}
+          topic={topic}
+          userErrorHandler={userErrorHandler}
+          userServiceAccount={userServiceAccount}
+          configString={configString}
+          configurationSteps={configurationSteps}
+        />
+      ) : (
+        <>
+          <dl className="pf-c-description-list pf-m-horizontal">
```

Comment:
Actually [use this one](https://patternfly-react-main.surge.sh/components/description-list#horizontal-using-custom-term-width-modifier) which will allow us to increase the term width to avoid needless wrapping.  Currently if I have the window maximized I see "Connector category" and "Connector name" wrapping, will post a screenshot.

Reply:
@gashcrumb - I like your suggestion to modify the width as shown in the second example you shared above. 

In a comment below, you said this:
> However I see some of the other fields can get lengthy as I'm re-reviewing the screenshots 😄 Maybe it'd be better to stick with the default width for now and ignore my comments around wrapping, sound good @Kells512?

And while I don't think it's possible (or desirable even) to avoid wrapping, I do think that with the length of the labels used in our Review step that a slightly wider width would give this a bit of breathing room. I also like how the widths are modified for different break points so that there's more space at larger viewports and it starts to shrink for smaller viewports.